### ISSUE TITLE
[Android 12] Bluetooth permissions

### ISSUE THREAD

**Issue by renefloor** [Baseflow/flutter-permission-handler#560]
## 🚀 Feature Requests

Android 12 introduces the BLUETOOTH_SCAN and BLUETOOTH_CONNECT permissions, which let your app scan for nearby devices without needing to request the location permission. Your app must target Android 12 to declare these permissions.

### Contextualize the feature
These are completely new Bluetooth Permissions for Android. These are not needed for iOS.

This can only be made/merged when the Android 12 platform is stable, probably in August.
![image](https://user-images.githubusercontent.com/15101411/118777104-fcb95700-b888-11eb-8231-60c0065149e4.png)

### Platforms affected (mark all that apply)
- [ ] :iphone: iOS
- [X] :robot: Android

**Comment by PieterAelse** [comment]
@renefloor  (CC @mvanbeusekom ) I've just opened https://github.com/Baseflow/flutter-permission-handler/pull/671 which does the work of this issue. Only thing left for you is to release the platform interface version and patch the import which is now using a `path`, see " # FIXME: Publish new version and use it here".

**Comment by PieterAelse** [comment]
@renefloor I think this issue can be closed 😄

**Comment by mvanbeusekom** [comment]
You are right, closing as this is fixed with #671 and published as version [8.2.0](https://pub.dev/packages/permission_handler/changelog#820) (also notice bug-fix in version [8.2.4](https://pub.dev/packages/permission_handler/changelog#824)).
